sql/driver/driver_test.go

```diff
@@ -149,6 +149,9 @@ func TestCreateTable(t *testing.T) {
 	if _, err := db.Exec("CREATE TABLE IF NOT EXISTS users " + cols); err != nil {
 		t.Fatal(err)
 	}
+	if _, err := db.Exec("CREATE TABLE employees " + cols); err != nil {
```

Comment:
Doesn't this still need a test that confirms that the session was indeed persisted on the connection between statements? in other words, I think we need to confirm that these databases were created on `t`.